tests/suite/fixtures.py

```diff
@@ -727,6 +733,64 @@ def fin():
     )
 
 
+class TransportServerSetup:
+    """
+    Encapsulate Transport Server Example details.
+
+    Attributes:
+        name (str):
+        namespace (str):
+    """
+
+    def __init__(self, name, namespace):
+        self.name = name
+        self.namespace = namespace
+
+
+@pytest.fixture(scope="class")
+def transport_server_setup(
+        request, kube_apis, test_namespace
+) -> TransportServerSetup:
+    """
+    Prepare Transport Server Example.
+
+    :param request: internal pytest fixture to parametrize this method:
+        {example: virtual-server|virtual-server-tls|..., app_type: simple|split|...}
```

Comment:
do the examples in the docs need to match TransportServer specific things?

Reply:
I don't think it adds value, so I've removed it entirely.